README.md

```diff
@@ -81,16 +81,9 @@ cron job would look like this:
 ###################
 5 8 * * 6 {/absolute/path/to/php} {/path/to/script/}sql-dump-sanitize.php --quiet
 
-# files directory backup #
+# Files directory backup #
 ##########################
-5 10 * * 6 {/absolute/path/to/php} {/path/to/script/}files-backup.php --rollover_files
+5 8 * * 6 {/absolute/path/to/php} {/path/to/script/}files-backup.php --rollover_files
```

Comment:
10:05am didn't seem to be referenced anywhere, so I assume this is also meant to be 8:05am...?